blueprints/ember-bootstrap/wizard.js

```diff
@@ -0,0 +1,162 @@
+    'app',
+    'components',
+    '*.js'
+  );
+  let files = fg.sync(componentsPath);
+  return files.map(file => path.basename(file, '.js'));
+}
+
+function buildComponentsChoices(config) {
+  let isChecked = component => {
+    return (
+      !config.whitelist ||
+      config.whitelist.length === 0 ||
+      config.whitelist.indexOf(component) !== -1
+    );
+  };
+
+  return getComponents().map(component => ({
+    name: component.replace('bs-', ''),
+    value: component,
+    checked: isChecked(component)
+  }));
+}
+
+function questions(options, config) {
+  let mergedConfig = Object.assign({}, config, options);
+  let guards = buildGuards(mergedConfig);
+
+  return [
+    {
+      name: 'customInstall',
+      message: 'Do you want to customize installation of ember-bootstrap?',
+      type: 'confirm',
+      default: false
+    },
+    {
+      name: 'bootstrapVersion',
+      message: 'Choose desired Bootstrap version:',
+      type: 'list',
+      default: BS_VERSIONS.indexOf(config.bootstrapVersion),
+      choices: BS_VERSIONS.map(v => ({ name: 'v' + v, value: v })),
+      when: guards(GUARD.customInstall, GUARD.noBS)
+    },
+    {
+      name: 'preprocessor',
+      message: 'Choose your CSS preprocessor:',
+      type: 'list',
+      default: PREPROCESSORS.indexOf(config.preprocessor),
+      choices(answers) {
+        let preprocessors = PREPROCESSORS;
+        if (answers.bootstrapVersion === 4) {
+          preprocessors = preprocessors.slice(0, -1);
+        }
+        return preprocessors;
+      },
+      when: guards(GUARD.customInstall, GUARD.noPreprocessor)
+    },
+    {
+      name: 'importBootstrapTheme',
+      message: 'Do you want to include the optional bootstrap-theme.css?',
+      type: 'confirm',
+      default: getDefault(config, 'importBootstrapTheme', false),
+      when: guards(GUARD.customInstall, GUARD.bs3, GUARD.noPreprocessor)
+    },
+    {
+      name: 'importBootstrapCSS',
+      message: 'Do you want to include the default static bootstrap.css?',
+      type: 'confirm',
+      default: getDefault(config, 'importBootstrapCSS', true),
+      when: guards(GUARD.customInstall, GUARD.noPreprocessor)
+    },
+    {
+      name: 'importBootstrapFont',
+      message: 'Do you want to include the Glyphicons font?',
+      type: 'confirm',
+      default: getDefault(config, 'importBootstrapFont', true),
+      when: guards(GUARD.customInstall, GUARD.bs3)
+    },
+    {
+      name: 'insertEmberWormholeElementToDom',
+      message:
+        "Do you want to automatically insert a div into your index.html, where Bootstrap's modals and tooltips will be rendered to.\n" +
+        chalk.red(
+          'If you set this to false, you will have to add an <div id="ember-bootstrap-wormhole"></div> element by yourself.'
+        ),
+      type: 'confirm',
+      default: getDefault(config, 'insertEmberWormholeElementToDom', true),
+      when: guards(GUARD.customInstall)
+    },
+    {
+      name: 'whitelist',
```

Comment:
The whitelist question is a bit problematic, when you have all components selected, as this will still write an array with all components to the config. Which is ok for the moment, but when new components get added, those will not be available to the user (as there is a whitelist without that new component), certainly causing a good amount of confusion. I *think* we briefly talked about this, was it on Slack?

I would prefer to have a way to skip writing `whitelist` (or writing `whitelist: null`) to capture the user's intend to not "tree shake" at all. What about a question first asking whether the user wants to tree-shake the available components? And only present the choices if he answered 'yes'?